packages/app-builder/src/routes/ressources+/auth+/refresh.tsx

```diff
@@ -41,7 +41,16 @@ export function useRefreshToken() {
           );
         },
         () => {
-          navigate(getRoute('/ressources/auth/logout'));
+          let redirectUrl = getRoute('/ressources/auth/logout');
+          if (window) {
+            const searchParams = new URLSearchParams();
+            searchParams.set(
+              'redirectTo',
+              `${window.location.pathname}${window.location.search}`,
+            );
+            redirectUrl = `${redirectUrl}?${searchParams.toString()}`;
```

Comment:
to be sure I totally understand, what happens here? If you're currently editing something and you get refreshed, you'd still loose your local state, wouldn't you ?

Reply:
Exactly. In fact it depends on the screen : for example, we use query params to store "filters state" in some views. Those screens will maintain "internal" state.

We could ser/de state to either :
- local storage
- client browser navigation state (not visible in the URL)
- specific query param + encoding

But I think this bring a lot of complexity due to load/unload that we do not want to maintain (at least for now)